mix.exs

```diff
@@ -21,8 +21,11 @@ defmodule Slack.Mixfile do
     [{:httpoison, "~> 0.11"},
      {:exjsx, "~> 3.2.0"},
      {:websocket_client, "~> 1.1.0"},
+
```

Comment:
Thats typically done to visually separate dev/prod dependencies. I'll remove if thats what you prefer